pkg/checks/checks.go

```diff
@@ -126,18 +242,10 @@ out:
 	}
 
 	// Statefulsets
-	ssList, err := appclient.StatefulSets("").List(metav1.ListOptions{})
-	if err != nil {
-		return err
-	}
-
-	for _, ss := range ssList.Items {
-		if !checkNamespace(ss.Namespace) {
-			continue
-		}
-		log.Info(fmt.Sprintf("checking %s %q", ss.Kind, ss.Name))
+	for idx, app := range statefulsetWhitelist {
+		log.Info(fmt.Sprintf("checking statefulset[%d]: [%s] %s", idx, app.Namespace, app.Name))
```

Comment:
log.Infof instead of log.Info+fmt.Sprintf

Reply:
Will fix.  thanks.